src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball() {
+        this.color = new ColorSupplier().getRandomColor();
+        this.number = new Random().nextInt(100);
```

Comment:
logic of getting of random color and number, should be move to Lottery's method, in constructor you should pass already formed parameters to initialize your object